### PR TITLE
PR Modified and added new batch of documentation

### PR DESCRIPTION
**Pull request by JasonKAEC**
PR concerns the following components:

- FormFieldSelectUserGroup.js
- FormFieldTags.js
- FormFieldTextArea.js
- FormFieldTitelEnBeschrijving.js
- FormFieldWerkingsgebied.js
- FormFieldWerkingsgebiedKoppeling.js
- FormFieldWerkingsgebiedrelatie.js
- GraphPopupMenu.js
- HeadingMain.js
- LeafletController.js
- LeafletDrawController.js
- LeafletRevisionOverview.js
- LeafletSearchInput.js
- LoaderBeleidsrelatieRegel.js
- LoaderCard.js
- LoaderCardHalfWidth.js
- LoaderContent.js
- LoaderMainTitle.js
- LoaderSaving.js
- LoaderSmallCircle.js
- LoaderSmallSpan.js
- LoaderSpinner.js
- LoaderWerkingsgebiedCard.js
- LoginForm.js
- MeldingAlgemeen/MeldingAlgemeen.js
- MeldingKoppeling/MeldingKoppeling.js

Features:
New batch of documentation and modified some comments.

### CODE REVIEW

File: src/components/FormFieldWerkingsgebiedKoppeling/FormFieldWerkingsgebiedKoppeling.js [hunk 1]
```diff
@@ -16,21 +16,17 @@ import PopupContainer from './../PopupContainer'
...
- * @param {string} titleSingular - Parameter that is passed down from the parent and contains the title value of the werkingsgebied object that is used in the id of a div within this component and its value is passed down to the FormFieldTitelEnBeschrijving component.
- * @param {string} fieldLabel - Parameter that is passed down from the parent and contains the field label value of the werkingsgebied object, which value is passed down to the FormFieldTitelEnBeschrijving component.
- * @param {string} pValue - Parameter that is passed down from the parent and contains the paragraf value of the werkingsgebied object, which value is passed down to the FormFieldTitelEnBeschrijving component.
- * @param {boolean} disabled - Parameter that is passed down from the parent, which value is passed down to the FormFieldTitelEnBeschrijving component and used in this component in a conditional operator of a div to set a certain style value.
+ * @param {function} werkingsgebiedInParentState - Parameter that contains the UUID of the werkingsgebied object.
+ * @param {string} dataObjectProperty - Parameter containing the werkingsgebied data property.
```
Comment (Aidenbuis) [inline src/components/FormFieldWerkingsgebiedKoppeling/FormFieldWerkingsgebiedKoppeling.js:25]: Parameter containing the werkingsgebied data property.
-> 
This field contains the property name for a geographical area, as this is not always the same on the crudObject.

File: src/components/FormFieldWerkingsgebiedKoppeling/FormFieldWerkingsgebiedKoppeling.js [hunk 2]
```diff
@@ -152,18 +148,17 @@ const FormFieldWerkingsgebiedKoppeling = ({
...
- * @param {function} setWerkingsgebiedInParentState - Parameter that is used to set the state value of the werkingsgebiedInParentState variable.
- * @param {function} setWerkingsgebied - Parameter that is used to set the state value of the werkingsgebied variable.
- * @param {object} dataObjectProperty - Parameter that is used to set the value within the target state value of the werkingsgebiedInParentState variable.
- * @param {object} werkingsgebied - Parameter used for conditional operators within the CardSelectedWerkingsgebied component, to display the title, a certain date format and the UUID for the url backgroundimage if it contains a value.
- * @param {boolean} show - Parameter to toggle the Transition component, that displays the CardSelectedWerkingsgebied component in a certain animation.
+ * @param {function} setPopupOpen - Parameter that is used set the state of the popupOpen variable.
+ * @param {function} setWerkingsgebiedInParentState - Function to edit parent state.
+ * @param {function} setWerkingsgebied - Function to edit parent state
+ * @param {object} dataObjectProperty - Parameter containing the werkingsgebied data property.
```
Comment (Aidenbuis) [inline src/components/FormFieldWerkingsgebiedKoppeling/FormFieldWerkingsgebiedKoppeling.js:159]: typeof string

File: src/components/FormFieldWerkingsgebiedKoppeling/FormFieldWerkingsgebiedKoppeling.js [hunk 3]
```diff
@@ -258,18 +253,15 @@ const CardSelectedWerkingsgebied = ({
...
  *
  * @function
  *
- * @param {boolean} show - Parameter if set true shows the PopupContainer component.
- * @param {boolean} close - Parameter if set true hides (closes) the PopupContainer component.
- * @param {object} setWerkingsgebiedInParentState - Parameter that is used to give value to the target variable and give it to the parent state.
- * @param {object} dataObjectProperty - Parameters value used in the condition statement and part of the value that is set within the setWerkingsGebiedInParentState object.
+ * @param {boolean} show - Parameter that is used to show the PopupContainer component.
+ * @param {boolean} close - Parameter that is called to close the PopupContainer component.
+ * @param {object} setWerkingsgebiedInParentState - Function to edit parent state.
```
Comment (Aidenbuis) [inline src/components/FormFieldWerkingsgebiedKoppeling/FormFieldWerkingsgebiedKoppeling.js:263]: typeof function

File: src/components/FormFieldWerkingsgebiedKoppeling/FormFieldWerkingsgebiedKoppeling.js [hunk 4]
```diff
@@ -258,18 +253,15 @@ const CardSelectedWerkingsgebied = ({
...
+ * @param {object} dataObjectProperty - containing the werkingsgebied data property.
```
Comment (Aidenbuis) [inline src/components/FormFieldWerkingsgebiedKoppeling/FormFieldWerkingsgebiedKoppeling.js:264]: typeof string

File: src/components/GraphPopupMenu/GraphPopupMenu.js [hunk 5]
```diff
@@ -50,6 +50,43 @@ const connectionPropertiesColors = {
     },
 }
 
+/**
```
Comment (Aidenbuis) [inline src/components/GraphPopupMenu/GraphPopupMenu.js:53]: These edits are preventable by rebasing your branch based on dev. Tip for the next time :)

File: src/components/HeadingMain/HeadingMain.js [hunk 6]
```diff
@@ -1,21 +1,12 @@
...
- * The component displays a div containing a h1 tag containing the titel variable (passed down from the parent) and uses the status variable in the
- * conditional operator to either display the status or not. Based on the isVigerend variable (which is set true if the status contains a certain value),
- * the color and border color of the status is set.
- *
- * This component is imported by the following component:
- * ContainerDetailMain
- *
- * This component is imported by the following page:
- * ContainerDetail
+ * Component that renders the HeadingMain component displaying a titel and status Vigerend or Gepubliceerd.
```
Comment (Aidenbuis) [inline src/components/HeadingMain/HeadingMain.js:4]: The status can also be something else then Vigerend / Gepubliceerd.

Component that renders the HeadingMain component displaying a titel and status Vigerend or Gepubliceerd.
->
Component that renders the HeadingMain component displaying a titel and status.

**Review by Aidenbuis — CHANGES_REQUESTED** [review]

File: src/components/FormFieldSelectUserGroup/FormFieldSelectUserGroup.js [hunk 7]
```diff
@@ -11,11 +11,11 @@ import LoaderSelect from './../LoaderSelect'
  *
  * @component
  *
- * @param {object} crudObject - Containing the ownership level of the user group.
- * @param {string} editStatus - Status of the selected user.
- * @param {boolean} handleChange - Can be set true or false if the user is allowed to change.
- * @param {string} titelEnkelvoud - Title of the selected user.
- * @param {boolean} disabled - Can be set true or false if the user is disabled or not.
+ * @param {object} crudObject - Contains the object that can be edited.
+ * @param {boolean} editStatus - Can be set True if
```
[the remark this answers is not in the document]
Comment (JasonKAEC) [reply src/components/FormFieldSelectUserGroup/FormFieldSelectUserGroup.js]: Thanks for the comment, I saw that I've forgotten to add text to that parameter! I've added it to the documentation.

File: src/components/FormFieldTags/FormFieldTags.js [hunk 8]
```diff
@@ -33,12 +32,11 @@ class FormFieldTags extends Component {
...
     /**
-     * Function to change the name and value variables.
+     * Function to handle change from the dropdown component from the FormFieldTags component.
      *
      * @function
      *
-     * @param {array} newValue - New array that will be set as a new value.
-     * @param {object} actionMeta - 001
+     * @param {array} newValue - Parameter used to set the value of the tags.
      */
```
[the remark this answers is not in the document]
Comment (JasonKAEC) [reply src/components/FormFieldTags/FormFieldTags.js:41]: Thanks for the comment, I saw that I've also forgotten to add that parameter! I've added it to the documentation.

File: src/components/FormFieldWerkingsgebiedKoppeling/FormFieldWerkingsgebiedKoppeling.js (hunk 1 above)
Comment (JasonKAEC) [reply src/components/FormFieldWerkingsgebiedKoppeling/FormFieldWerkingsgebiedKoppeling.js]: Thanks for the description change for the parameter. I've changed the description accordingly.

File: src/components/FormFieldWerkingsgebiedKoppeling/FormFieldWerkingsgebiedKoppeling.js (hunk 3 above)
Comment (JasonKAEC) [reply src/components/FormFieldWerkingsgebiedKoppeling/FormFieldWerkingsgebiedKoppeling.js]: I've changed the typeOf from object ==> function for the setWerkingsgebiedInParentState parameter.

File: src/components/FormFieldWerkingsgebiedKoppeling/FormFieldWerkingsgebiedKoppeling.js (hunk 2 above)
Comment (JasonKAEC) [reply src/components/FormFieldWerkingsgebiedKoppeling/FormFieldWerkingsgebiedKoppeling.js]: I've changed the typeOf from object ==> string for the dataObjectProperty parameter.

File: src/components/FormFieldWerkingsgebiedKoppeling/FormFieldWerkingsgebiedKoppeling.js (hunk 4 above)
Comment (JasonKAEC) [reply src/components/FormFieldWerkingsgebiedKoppeling/FormFieldWerkingsgebiedKoppeling.js]: I've changed the typeOf from object ==> string for the dataObjectProperty parameter.

File: src/components/GraphPopupMenu/GraphPopupMenu.js (hunk 5 above)
Comment (JasonKAEC) [reply src/components/GraphPopupMenu/GraphPopupMenu.js]: Thanks for the tip! Apparently my rebase didn't work properly haha, but I'll make sure that the rebase is successful next time.

File: src/components/HeadingMain/HeadingMain.js (hunk 6 above)
Comment (JasonKAEC) [reply src/components/HeadingMain/HeadingMain.js]: That makes sense, thanks for the explanation. And I've changed the description and removed the "Vigerend or Gepubliceerd" text.

**Pull request merged**